### ISSUE TITLE
Plugin reporting plays for both "is playing" and "has finished playing"

### ISSUE THREAD

**Issue by mordac** [jellyfin/jellyfin-plugin-trakt#86]
Any time I play a show and it's the end of a playlist (nothing plays after), I get two plays on trakt, one from when I started playing and one when finished. This doesn't happen if there's a file played after, as in for every episode before the last.

Steps to reproduce:

1. Play the last file of a show or play a movie
2. Let play to 100%

Expected outcome:

Episode/movie has 1 play reported

Actual outcome:

Episode has two plays as if watched twice.

[Here's an excerpt of my log that I hope contains relevant information. I can't post the whole thing because it's too big.](https://pastebin.com/Z747ATD4)

It seems to report 

`[INF] [13] Trakt.ServerMediator: Playback Started`

every time a file is played, and then 

```
[INF] [61] Trakt.ServerMediator: Playback Stopped
[INF] [61] Trakt.ServerMediator: Item is played. Scrobble
```

only when a file is 100% played and stops.

**Comment by davidfrickert** [comment]
I also seem to have this problem. Just realized that i have some episodes with alot of plays when im pretty sure i've only watched them once.
But most likely alot of start / stopping. And likely that each stop even if the episode wasn't fully watched it's reporting as a play? Which seems weird

**Comment by devential** [comment]
I'd like to add my own observations for this: both the start and end of the play are recorded, e.g., if I start an episode at 2:00 and it finishes at 2:30, I get records for both 2 and 2:30 on Trakt. This also occurs for any of the clients I use as long as the video plays through to the end (Kodi plugin, Android client, Jellyfin Media Player, browser).

**Comment by Jerrk** [comment]
i think my issue is the same, trakt is recording multiple plays even though the episode is only played once.

![](https://i.imgur.com/2RigPHA.png)
![](https://i.imgur.com/0wyNH4G.png)

only sometimes though which is odd, i watched the whole series and it only reported multiple plays for episodes 1, 2, and 8.

**Comment by Shadowghost** [comment]
According to [the trakt API docs](https://trakt.docs.apiary.io/#reference/scrobble/stop/stop-or-finish-watching-in-a-media-center) an item is scrobbled (marked as played) when you stop playback at over 80% (so in the case of Jellyfin at 100%) and then again when Jellyfin internally marks it as played. The workaround should be to call the pause endpoint instead of the stop endpoint whenever Jellyfin stops playback.

**Comment by Shadowghost** [comment]
Should be fixed with #120 which only calls stopped if jellyfin internally marks it as watched.
